newdle/providers/free_busy/exchange.py

```diff
@@ -46,8 +73,13 @@ def fetch_free_busy(date, uid):
         ),
     ]
 
-    tz = uid_account.default_timezone
-    start = utc.localize(datetime.combine(date, time.min)).astimezone(tz)
+    if tz in NON_STANDARD_TZS:
```

Comment:
tz = NON_STANDARD_TZS.get(tz, tz)

Reply:
Yeah, thought of it but it looked dirtier that way... "explicit better than implicit" etc...

Reply:
+1 to more explicit code here